src/main/java/net/dries007/tfc/common/blocks/wood/TFCSaplingBlock.java

```diff
@@ -31,9 +32,14 @@
 public class TFCSaplingBlock extends SaplingBlock implements IForgeBlockExtension, EntityBlockExtension
 {
     private final ExtendedProperties properties;
-    private final int daysToGrow;
+    private final Supplier<Integer> daysToGrow;
```

Comment:
```suggestion
    private final IntSupplier daysToGrow;
```

Reply:
this is incompatible with config values provided by forge

Reply:
Yeah, but you already coerce it by passing in `this::daysToGrow`. You should be able to change the type of this without issue.

In either case, it's not a huge issue and is fine to leave as-is. Also lets addons pass in `configValue` instead of `configValue::get`. So sure, why not.